timesketch/lib/tasks.py

```diff
@@ -17,6 +17,7 @@
 import subprocess
 import traceback
 
+from celery import Task
```

Comment:
You don't seem to be using this anywhere in the code. Correct?

Reply:
Good catch! Removed now.